var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -90,6 +90,8 @@ class Qt(Package):
     patch('qt4-pcre-include-conflict.patch', when='@4')
     patch('qt4-el-capitan.patch', when='@4')
 
+    patch('qt4-gcc-and-webkit.patch', when='@4')
```

Comment:
can you add a comment what the patch is doing and point to the issues/PRs on github?